src/analytics/ValoraAnalytics.ts

```diff
@@ -25,7 +25,36 @@ import { Statsig } from 'statsig-react-native'
 
 const TAG = 'ValoraAnalytics'
 
-async function getDeviceInfo() {
+interface DeviceInfoType {
```

Comment:
ah, how annoying that there's no type definition from the lib itself 🙈

Reply:
there is actually